sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryStatusAndProgressSuite.scala

```diff
@@ -38,13 +38,18 @@ class StreamingQueryStatusAndProgressSuite extends SparkFunSuite {
         |  "id" : "${testProgress1.id.toString}",
         |  "runId" : "${testProgress1.runId.toString}",
         |  "name" : "myName",
-        |  "timestamp" : "2016-12-05T20:54:20.827Z",
+        |  "triggerTimestamp" : "2016-12-05T20:54:20.827Z",
```

Comment:
@marmbrus One idea is to not have a top level timestamp, and merge this with the `queryTimestamps`. Then this would be a key `triggerStartTime` in the `queryTimestamps` map. In fact then we can rename `queryTimestamps` to `timestamps`.